src/Mvc/samples/MvcSandbox/Startup.cs

```diff
@@ -67,8 +67,6 @@ public void Configure(IApplicationBuilder app)
                     });
 
                 builder.MapApplication();
-
-                builder.MapHealthChecks("/healthz");
```

Comment:
MapHealthChecks here was a test example. Removed because it isn't needed anymore and it conflict with the real extension method.